src/darwin/Framework/CHIP/CHIPDevice.mm

```diff
@@ -1088,6 +1105,11 @@ - (void)subscribeAttributeWithEndpointId:(NSUInteger)endpointId
         chip::app::ReadPrepareParams readParams([self internalDevice]->GetSecureSession().Value());
         readParams.mpAttributePathParamsList = container.pathParams;
         readParams.mAttributePathParamsListSize = 1;
+        readParams.mMinIntervalFloorSeconds = static_cast<uint16_t>([minInterval unsignedShortValue]);
+        readParams.mMaxIntervalCeilingSeconds = static_cast<uint16_t>([maxInterval unsignedShortValue]);
+        readParams.mIsFabricFiltered = (params == nil || params.fabricFiltered == nil || [params.fabricFiltered boolValue]);
+        readParams.mKeepSubscriptions
+            = (params == nil || params.keepPreviousSubscriptions == nil || [params.keepPreviousSubscriptions boolValue]);
```

Comment:
Again, this is defaulting to YES when it should default to NO for mKeepSubscriptions.